AmplicaLabs/social-app-template
# 61 update backend readme

# Purpose
The goal of this PR is to update the readme using the template.

Closes #61 

## Solution

Used the template to fill out the readme.


### Change summary
* Used the template and previous readme content to fill out the readme.


## Steps to Verify
1. Make sure that if you follow the steps it works
2. Make sure that it's as readable and organized as possible
